src/site/content/en/blog/idle-detection/index.md

```diff
@@ -0,0 +1,238 @@
+The origin trial is expected to end in Chrome TODO.
+
+{% include 'content/origin-trials.njk' %}
+
+### Register for the origin trial {: #register-for-ot }
+
+{% include 'content/origin-trial-register.njk' %}
+
+### Feature detection
+
+To check if the Idle Detection API is supported, use:
+
+```javascript
+if ('IdleDetector' in window) {
+  // Idle Detector API supported
+}
+```
+
+### Idle Detection API concepts
+
+The Idle Detection API assumes that there is some level of engagement between the user,
+the user agent (that is, the browser), and the operating system of the device in use.
+This is represented in two dimensions:
+
+- **The user idle state:**
+  `active` or `idle`: the user has, respectively has not,
+interacted with the user agent for some period of time.
+- **The screen idle state:**
+  `locked` or `unlocked`: the system has an active screen lock (like a screensaver) preventing
+  interaction with the user agent.
+
+Distinguishing `active` from `idle` requires heuristics that may differ across user, user agent,
+and operating system. It should also be a reasonably coarse threshold
+(see [Security and Permissions](#security-and-permissions)).
+
+The model intentionally does not formally distinguish between interaction with particular content
+(that is, the webpage in a tab using the API), the user agent as a whole, or the operating system;
+this definition is left to the user agent.
+
+### Using the Idle Detection API
+
+The first step when using the Idle Detection API is to initialize the `IdleDetector`.
+It takes an object with the desired idle `threshold` in seconds.
+The minimum `threshold` is 60 seconds. You start the idle detection by calling the
+`IdleDetector`'s `start()` method, and you can stop it again by calling the `stop()` method.
+
+There are two ways to get updates on the user's idle state: the first is for one-off requests by
+checking the `IdleDetector`'s `state` property. The sample below polls the `state`,
+which after one minute of inactivity will be reported as the `user` being `"idle"`.
+
+```js
+try {
+  const idleDetector = new IdleDetector({ threshold: 60 });
+  await idleDetector.start();
+  console.log("Initial idle state:", idleDetector.state);
+  // Every 10 seconds poll the user's idle state
+  const interval = setInterval(() => {
+    console.log("Current idle state:", idleDetector.state);
+  }, 10 * 1000);
+  // After 2min, stop the idle detection
+  setTimeout(async () => {
+    clearInterval(interval);
+    await idleDetector.stop();
+  }, 120 * 1000);
+} catch (err) {
+  // Deal with initialization errors like
+  // permission denied, running outside of top-level frame, etc.
+  console.error(err.name, err.message);
+}
+```
+
+The second, _and preferred_, way is by subscribing to `change` events via an event listener.
+To extract the two properties of interest of the fired event, namely `user` and `screen`,
+you can use the new object destructuring syntax in JavaScript.
```

Comment:
I'm not seeing any object destructuring syntax in the code sample?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring

Reply:
Good catch. This was simplified, since it added confusion. The sentence remained, though.